### ISSUE TITLE
Feature Request: Generate UUID

### ISSUE THREAD

**Issue by bassey3** [IvanMathy/Boop#77]
A simple Version 4 UUID Generator.

**Comment by IvanMathy** [comment]
Agreed, I needed it a while ago but it turned out complicated to do. Once #42 gets through it's gonna be a lot easier.

**Comment by 4brunu** [comment]
I also need this.
I saw two PRs that were rejected.
Any news on this?
